### PR TITLE
Closes #14 Update README with how to use icons.

### LINKED ISSUE #14: Update the readme AZ Icon reference link after a release is made.
## Motivation
Right now, the readme has links pointing to main instead of a versioned copy of az-icons.

Once a release is made, the readme will need updating.

### PR DESCRIPTION
**Pull request by trackleft**
Update readme with how to use the icons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,90 @@
...
+When choosing which version to include in your project, take these three choices
+into account:
+
+**Stable** version: This option is the most recent version that has the most
+wide-spread adoption on sites.
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/1.0.0/az-icons-styles.css" crossorigin="anonymous">
+```
+
+**Latest** version: Latest tagged version. The most recently tagged stable
```
Comment (akslay) [inline README.md:25]: Could the use of "stable" here be confusing when comparing to the "stable version"?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,90 @@
...
+release of AZ Bootstrap. While this has not been explicitly tested on this
+version of az_barrio, it’s probably OK to use on production sites. Please
```
Comment (akslay) [inline README.md:27]: What version is "this version of az_barrio"?

Comment (trackleft) [inline README.md:27]:
```suggestion
release of AZ Bootstrap. While this has not been tested extensively out in the wild,
it’s probably OK to use on production sites. Please
```

File: README.md (hunk 1 above)
Comment (trackleft) [reply README.md:25]: I'm not sure, what about:

```
Latest tagged version. This contents of this CDN link update automatically, every time we tag a release of `az-icons`. While there is some risk that if you use this link, the icons could randomly break on your site, the up-side is that you get access to new icons and upgrades to performance on each release.
```

File: README.md [hunk 3]
```diff
@@ -1,32 +1,90 @@
...
-<link href="https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css" rel="stylesheet">
+### Arizona Digital CDN
+Skip the download with the Arizona Digital CDN to deliver cached version of
+Arizona Icons’s CSS and font files to your project.
+
+When choosing which version to include in your project, take these three choices
+into account:
+
+**Stable** version: This option is the most recent version that has the most
+wide-spread adoption on sites.
```
Comment (trackleft) [inline README.md:20]:
```suggestion
**Stable** version: Each tagged release has a permanent CDN link. This version has to be upgraded on your project manually.
```

**Review by akslay — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,32 +1,92 @@
...
-## Using the Icons
-The icons can be used with Arizona Digital CDN link:
+### Source Files
+Compile Arizona Icons with your own asset pipeline by downloading our source
+CSS, SVG, PNG, and font files. This option requires some additional tooling:
 
-```html
-<link href="https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css" rel="stylesheet">
+### Arizona Digital CDN
+Skip the download with the Arizona Digital CDN to deliver cached version of
```
Comment (bberndt-uaz) [inline README.md:13]:
```suggestion
Skip the download with the Arizona Digital CDN to deliver a cached version of
```

File: README.md [hunk 5]
```diff
@@ -1,32 +1,92 @@
...
+
+**Stable** version: Each tagged release has a permanent CDN link. This version
+has to be upgraded on your project manually.
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/1.0.0/az-icons-styles.css" crossorigin="anonymous">
+```
+
+**Latest** tagged version: The contents of this CDN link update automatically
+every time we tag a release of `az-icons`. While there is some risk involved
+with this link, for example, the icons could randomly break on your site, the
```
Comment (bberndt-uaz) [inline README.md:27]:
```suggestion
with this link (for example, the icons could randomly break on your site), the
```

File: README.md [hunk 6]
```diff
@@ -1,32 +1,92 @@
...
+up-side is that you get access to new icons and upgrades to performance on each
+release automatically. It's probably OK to use on production sites. Please
+report bugs to the AZ Digital team.
```
Comment (bberndt-uaz) [inline README.md:30]: Would it be good to link "report bugs" to the Issues page or another page? The same report bugs link could be added to the Main version text below, or perhaps that note could be moved elsewhere.

File: README.md [hunk 7]
```diff
@@ -1,32 +1,92 @@
...
+**Main** version: Latest dev version. This is the tip of the main branch of
+AZ Icons. Please do not use on production unless you are following the AZ
+Icons project closely. Please report bugs to the AZ Digital team.
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css" crossorigin="anonymous">
 ```
 
-*(Latest vs. Pinned release information coming soon)*
+### Package Managers
+Pull in Arizona Icons’ source files into nearly any project with some of
```
Comment (bberndt-uaz) [inline README.md:43]:
```suggestion
Pull the Arizona Icons source files into nearly any project with some of
```

File: README.md [hunk 8]
```diff
@@ -1,32 +1,92 @@
...
-- [ ] Do not use the automatically generated CSS file from IcoMoon, instead only add the new icon class declaration (the default stylesheet provided by IcoMoon has been altered to enhance usage).
\ No newline at end of file
+- [ ] The original PUA codes need to be matched to their corresponding icons in
+  IcoMoon (a newly introduced icon will need an unused PUA code specified).
+- [ ] Do _not_ change or duplicate the PUA codes connected with each existing
+  icon (i.e., the icon class `.az-icon-award` is currently mapped to the PUA
+  code `e901`).
+- [ ] Make sure to name the font **az-icons** before clicking IcoMoon App's font
+  download button (the name can be set by clicking the small gear icon next to
+  the download button first)
```
Comment (bberndt-uaz) [inline README.md:89]:
```suggestion
  the download button first).
```

File: README.md (hunk 6 above)
Comment (trackleft) [inline README.md:30]:
```suggestion
[report bugs to the AZ Digital team](https://github.com/az-digital/az-icons/issues/new?assignees=&labels=&template=bug_report.md).
```

File: README.md [hunk 9]
```diff
@@ -1,32 +1,92 @@
...
+up-side is that you get access to new icons and upgrades to performance on each
+release automatically. It's probably OK to use on production sites. Please
+[report bugs to the AZ Digital team](https://github.com/az-digital/az-icons/issues/new?assignees=&labels=&template=bug_report.md).
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/latest/az-icons-styles.css" crossorigin="anonymous">
+```
+
+**Main** version: Latest dev version. This is the tip of the main branch of
+AZ Icons. Please do not use on production unless you are following the AZ
+Icons project closely. Please report bugs to the AZ Digital team.
```
Comment (trackleft) [inline README.md:37]:
```suggestion
Icons project closely. Please [report bugs to the AZ Digital team](https://github.com/az-digital/az-icons/issues/new?assignees=&labels=&template=bug_report.md).
```

**Review by bberndt-uaz — DISMISSED** [review]

File: README.md [hunk 10]
```diff
@@ -1,32 +1,92 @@
...
+Bootstrap Docs](https://digital.arizona.edu/arizona-bootstrap/docs/2.0/icons/).
+## Download
+Download Arizona Icons to get the compiled CSS and font files, source code, or
+include it with your favorite package managers like npm, RubyGems, and more.
 
-## Using the Icons
-The icons can be used with Arizona Digital CDN link:
+### Source Files
+Compile Arizona Icons with your own asset pipeline by downloading our source
+CSS, SVG, PNG, and font files. This option requires some additional tooling:
```
Comment (camikazegreen) [inline README.md:10]:
```suggestion
CSS, SVG, PNG, and font files. This option requires some additional tooling.
```

File: README.md [hunk 11]
```diff
@@ -1,32 +1,92 @@
...
+has to be upgraded on your project manually.
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/1.0.0/az-icons-styles.css" crossorigin="anonymous">
+```
+
+**Latest** tagged version: The contents of this CDN link update automatically
+every time we tag a release of `az-icons`. While there is some risk involved
+with this link (for example, the icons could randomly break on your site), the
+up-side is that you get access to new icons and upgrades to performance on each
+release automatically. It's probably OK to use on production sites. Please
```
Comment (camikazegreen) [inline README.md:29]:
```suggestion
release automatically. It's safe to use on production sites. Please
```

**Review by camikazegreen — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]

**Review by cpoff — APPROVED** [review]

**Pull request merged**